test/unittest/tests/test_extent_list.cpp

```diff
@@ -227,8 +243,59 @@ TEST_CASE("Extent Lists: block_off_srch_last", "[extent_list]")
         for (int i = 1; i < WT_SKIP_MAXDEPTH; i++)
             head.push_back(second->_raw);
 
-        REQUIRE(__ut_block_off_srch_last(&head[0], &stack[0]) == second->_raw);
+        for (int i = 0; i < WT_SKIP_MAXDEPTH; i++)
+            el.off[i] = head[i];
+
+        REQUIRE(__ut_block_off_srch_last(&el, &stack[0], true) == second->_raw);
+    }
+}
+
+TEST_CASE("Extent Lists: block_off_srch_last_for_mock_session", "[extent_list]")
+{
+    std::shared_ptr<MockSession> mock_session;
+    WT_EXTLIST *alloc;
+    wt_off_t off, size;
+    WT_EXT *ext, extp, **astack[WT_SKIP_MAXDEPTH];
+    WT_BLOCK *block;
```

Comment:
These types should all be in alphabetical order.

Reply:
I have change the code,  Is there documentation for coding specifications? If there is, I'll study it.
 Thank you

Reply:
It is documented at https://github.com/wiredtiger/wiredtiger/blob/develop/CONTRIBUTING.rst

Reply:
Following your instructions, I have change the code.
thanks.